### PR TITLE
Davdar patch 2

### PR DESCRIPTION
**Pull request by davdar**

### CODE REVIEW

File: monadic-eval/paper2/01-intro.tex [hunk 1]
```diff
@@ -15,22 +15,22 @@ \section{Introduction}
 The essence of the state-machine based approach was distilled by Van
 Horn and Might in their ``abstracting abstract machines'' (AAM)
 technique, which provides a systematic method for constructing
-abstract interpreters from standard abstract machines like the CEK- or
```
Comment (dvanhorn) [inline monadic-eval/paper2/01-intro.tex]: This was correct as is.  It's saying CEK-machine or Krivine-machine in shorthand way: CEK- or Krivine-machine.

Comment (davdar) [reply monadic-eval/paper2/01-intro.tex]: ah, I see

File: monadic-eval/paper2/01-intro.tex [hunk 2]
```diff
@@ -50,20 +50,20 @@ \section{Introduction}
 The second regard is more suprising, and we consider its observation
 to be the main contribution of this pearl.
 
-Definitional interpreters, compared with abstract machines, can leave
+Definitional interpreters, as opposed to abstract machines, can leave
```
Comment (dvanhorn) [inline monadic-eval/paper2/01-intro.tex]: How about "in contrast to" instead?

Comment (davdar) [reply monadic-eval/paper2/01-intro.tex]: I like it

Comment (dvanhorn) [reply monadic-eval/paper2/01-intro.tex]: Can you update the pull request?

Comment (davdar) [reply monadic-eval/paper2/01-intro.tex]: yep. I'll revert the CEK- thing too.

**Comment by davdar** [comment]
updated.

**Pull request merged**